pennylane/qchem/molecule.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         self.mult = mult
         self.basis_name = basis_name.lower()
         self.name = name
+        self.load_data = load_data
```

Comment:
We could add a line to `basis_name` docstring about using `load_data` argument when they want to use a basis set other than the ones specified.

Reply:
Added.